plugins/vendor/webex/batch/processors/KWebexAPIDropFolderEngine.php

```diff
@@ -34,11 +34,18 @@ class KWebexAPIDropFolderEngine extends KDropFolderFileTransferEngine
 	public function watchFolder(KalturaDropFolder $dropFolder)
 	{
 		$this->initDropFolderEngine($dropFolder);
-		$recordingsList = $this->retrieveRecordingsList();
-		if ($recordingsList)
+		$paginationLink = null;

```

Comment:
i would call it nextPageLink (unless its a fixed one which i assume its not the case)